docs/runbook/Quick_Start.md

```diff
@@ -56,12 +57,22 @@ python3 -m unittest discover terragraph_planner -b
 
 ## Run a Plan
 
+### Configuration File
+
+To customize and run a plan, you will need an input configuration yaml file.
```

Comment:
Since configuration yaml file is not the only way (but yes, it's the easiest way) to run a plan, I wonder "will need" are good words here.